library/Zend/ServiceMananger/Proxy/LazyServiceFactory.php

```diff
@@ -22,17 +21,17 @@
  *
  * @link https://github.com/Ocramius/ProxyManager/blob/master/docs/lazy-loading-value-holder.md
  */
-class LazyServiceFactory implements DelegatorFactoryInterface
+final class LazyServiceFactory implements DelegatorFactoryInterface
```

Comment:
This is a factory, I do not think you actually need to extend it. And if you need, you should use a delegator.

As ProxyManager may evolve, declaring it final may actually allows us to do breaking changes inside the code without having to release a new major release.